reader/scraper/rules.go

```diff
@@ -7,6 +7,7 @@ package scraper // import "miniflux.app/reader/scraper"
 // List of predefined scraper rules (alphabetically sorted)
 // domain => CSS selectors
 var predefinedRules = map[string]string{
+	"feeds.bbci.co.uk":    "div.vxp-column--single, div.story-body__inner, ul.gallery-images__list",
```

Comment:
You should use `bbc.co.uk` here, because this is the domain used by entry URLs, otherwise this rule won't be taken into consideration.